src/main/java/mrjake/aunis/tileentity/stargate/StargateUniverseBaseTile.java

```diff
@@ -42,105 +42,103 @@
 
 	// --------------------------------------------------------------------------------
 	// Dialing
-	
```

Comment:
It's recommended to do code optimisations in different commits/PRs. Because it's a bit harder to review PR with this

Reply:
I tried to keep these files as they were, but IntelliJ doesn't have an option to disable formatting on save, sorry for that.

Reply:
Yes it does. If you're making commits via Intellij Idea there is checkmark `Do you want to optimise code before commit?`. And there's no automatic optimisations on save unless you turned this on by yourself. Check IDE settings and look at this checkmark on commit

Reply:
I'm not doing commits via intellij, but it's formatting on save, which doesn't seem to have an option to disable that.

Reply:
I've read there is a macro bound to Ctrl+S by default in intellij